org.eclipse.cpacep/src/org/eclipse/cpacep/test/CPACEPConnectorTest.java

```diff
@@ -0,0 +1,53 @@
+package org.eclipse.cpacep.test;
```

Comment:
Tests should be in the same package as the classes that they test.
This makes it easy to find the tests for a class, and allows you to use and access package-private methods for testing.